ambari-server/src/main/resources/common-services/YARN/2.1.0.2.0/configuration/container-executor.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<configuration supports_adding_forbidden="true">
+  <property>
+    <name>content</name>
+    <display-name>container-executor configuration template</display-name>
+    <description>This is the jinja template for container-executor.cfg file</description>
+    <property-type>VALUE_FROM_PROPERTY_FILE</property-type>
+    <value/>
+    <value-attributes>
+	  <type>content</type>
+      <property-file-name>container-executor.cfg.j2</property-file-name>
+      <property-file-type>text</property-file-type>
+    </value-attributes>
+    <on-ambari-upgrade add="true"/>
```

Comment:
@dlysnichenko , @smolnar82 , doesn't this need to be set to false?

Reply:
yes, it should be false

Reply:
fixed